app/views/activities/_filters.html.erb

```diff
@@ -0,0 +1,14 @@
+<div class="activity-filters ">
+  <%= form_with url: project_activities_path(current_project.id), method: :get, remote: true, class: 'form-row d-flex flex-row' do |f| %>
+    <%= f.select :user, options_from_collection_for_select(User.all, :id, :name, params[:user]), { include_blank: 'Filter by user'}, { class: 'form-select m-2 form-select-sm', onchange: '$(this.form).submit();' } %>
+    <%= f.select :type, Activity.select(:trackbable_type).distinct.pluck(:trackable_type), { include_blank: 'Filter by type', selected: params[:type]}, { class: 'form-select m-2 form-select-sm', onchange: '$(this.form).submit();' } %>
+
+    <div id="" class="input-group">
+      <%= f.date_field :period_start, max: Date.today, placeholder: "From date", value: params[:period_start], class: 'form-control form-control-sm my-2', onchange: '$(this.form).submit();' %>
```

Comment:
Let's add a span here that looks like a label, I think `Date range` will work well. Then we need some actual label elements for both date inputs that have a `visually-hidden` class for accessibility.

Reply:
Something like this?
<img width="974" alt="Screenshot 2023-12-06 at 14 05 18" src="https://github.com/dradis/dradis-ce/assets/2478434/48ec10dc-135a-4809-81d9-594a38f147a9">